README.md

```diff
@@ -73,28 +54,55 @@ builder.Services.AddSingleton<ISinch>(x => new SinchClient(
     builder.Configuration["Sinch:ProjectId"]));
 ```
 
+To configure Conversation or Sms hosting regions, and any other additional paramters, use [`SinchOptions`](https://github.com/sinch/sinch-sdk-dotnet/blob/main/src/Sinch/SinchOptions.cs):
+
+```csharp
+var sinch = new SinchClient(
+    configuration["Sinch:KeyId"],
+    configuration["Sinch:KeySecret"],
+    configuration["Sinch:ProjectId"],
+    options =>
+    {
+        options.SmsHostingRegion = Sinch.SMS.SmsHostingRegion.Eu;
+        options.ConversationRegion = Sinch.Conversation.ConversationRegion.Eu;
+    });
+```
 
-## Products
+## Supported Sinch Products
 
 Sinch client provides access to the following Sinch products:
 
-- Numbers
-- SMS
-- Verification
-- Voice
-- Work-in-Progress Conversation
+- [SMS](https://developers.sinch.com/docs/sms/)
+- [Conversation](https://developers.sinch.com/docs/conversation/)
+- [Numbers](https://developers.sinch.com/docs/numbers/)
+- [Verification](https://developers.sinch.com/docs/verification/)
+- [Voice](https://developers.sinch.com/docs/voice/)
+- additional products coming soon!
+
+## Logging, HttpClient, and additional options
 
-Usage example of the `numbers` product, assuming `sinch` is a type of `ISinchClient`:
+To configure logger, provide own `HttpClient`, and additional options utilize `SinchOptions` within constructor:
```

Comment:
This sentence is not syntactically correct.
Suggestion (to be adapted by someone fluent with .NET terminology 😅): "To configure a logger, provide your own `HttpClient` or any additional supported options, you can use the `SinchOptions` Action defined by the `options` property in the constructor:"